src/pages/terraform/downloads/index.tsx

```diff
@@ -30,33 +34,34 @@ const TerraformDownloadsPage = (props: GeneratedProps): ReactElement => {
 }
 
 /**
- * Pulled from terraform-website/pages/downloads/index.jsx on 3/9/2022:
+ * Pulled from terraform-website/pages/downloads/index.jsx on 2022-03-09:
  * https://github.com/hashicorp/terraform-website/blob/master/pages/downloads/index.jsx#L55-L98
+ *
+ * Modified 2022-03-28 to replace semverGte with semverSatisfies,
+ * as the former seemed to return pre-releases, which is not what we want.
  */
-function filterOldVersions(props) {
-  if (!props?.props?.releases?.versions) {
-    return props
-  }
-
-  const versions = props.props.releases.versions
-
-  // versions is in the form of { [version]: { ...metadata } }
+function filterVersions(
```

Comment:
Refactored to avoid modifying `props.props.releases.versions` as a kind of side-effect. Now accepts versions, and returns filteredVersions.